src/integrationTest/resources/application.yaml

```diff
@@ -55,9 +55,9 @@ doc_assembly:
 docmosis:
   templates:
     english:
-      DIVORCE_MINI_PETITION: Divorce_CP_Mini_Draft_Petition_Template.docx
+      DIVORCE_MINI_APPLICATION: Divorce_CP_Mini_Draft_Application_Template.docx
     welsh:
-      DIVORCE_MINI_PETITION: FL-DIV-GNO-WEL-00256.docx
+      DIVORCE_MINI_APPLICATION: FL-DIV-GNO-WEL-00256.docx
```

Comment:
do we need to change the document wording as well?

Reply:
Good point, let me raise a separate PR for that document. 👍